apps/kaizen/api/v1/serializers.py

```
@@ -1,17 +1,198 @@
+
+        is_highlight = validated_data.get("is_highlight", False)
+
+        if is_highlight and not instance.is_highlight:
+            due_date = validated_data.get("due_date", instance.due_date)
+
+            if due_date:
+                self._clear_existing_highlights(due_date, instance.id)
+
+        return super().update(instance, validated_data)
+
+    def _clear_existing_highlights(self, date, exclude_id=None):
```

Ruff (0.8.2)

Missing return type annotation for private function `_clear_existing_highlights`

Add return type annotation: `None`

(ANN202)